src/components/TheDrawer.vue

```diff
@@ -0,0 +1,66 @@
+<script setup lang="ts">
+import { Drawer, DrawerContent } from "@progress/kendo-vue-layout";
+import { useLocalStorage } from "@vueuse/core";
+import { useRouter } from "vue-router";
+import { computed, ref } from "vue";
+// const router = useRouter();
```

Comment:
you can remove it

Reply:
True, at this moment only useLocalStorage and computed are being used